src/translate_rapidpro_keys.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime
+
+import pytz
+from core_data_modules.traced_data import Metadata
+from core_data_modules.util import TimeUtils
+from dateutil.parser import isoparse
+
+
+class TranslateRapidProKeys(object):
+    # TODO: Move the constants in this file to configuration json
+    SHOW_ID_MAP = {
```

Comment:
Can you remove these two constants if not using them?

Reply:
Removed